gradle.properties

```diff
@@ -26,9 +26,11 @@ intellijPluginVersion=1.1.2
 javassistVersion=3.27.0-GA
 kotlinVersion=1.5.20
 mockitoKotlinVersion=3.2.0
-projectorClientVersion=021a2cda
-projectorClientGroup=com.github.JetBrains.projector-client
+projectorClientVersion=fa179d0b
+projectorClientGroup=org.jetbrains.projector
```

Comment:
I suggest changing the group only on CI. Please see https://github.com/JetBrains/projector-server/pull/63 with the fix